src/solver/optim-model-filler/include/antares/solver/optim-model-filler/TimeDependentLinearExpression.h

```diff
@@ -59,9 +59,12 @@ class TimeDependentLinearExpression
     TimeDependentLinearExpression operator-() const;
     TimeDependentLinearExpression shiftLinearExpressions(int shiftValue) const;
     TimeDependentLinearExpression operator[](int index) const;
```

Comment:
```c++
TimeDependentLinearExpression operator[](int index) const
```
the argument is a time step, not a fuzzy index.
We should indicate it : `TimeDependentLinearExpression operator[](int timeStep) const`